src/loaddb/load_common.cpp

```diff
@@ -537,7 +538,8 @@ namespace cubload
   }
 
   int
-  split (int batch_size, const std::string &object_file_name, class_handler &c_handler, batch_handler &b_handler)
+  split (int batch_size, const std::string &object_file_name, class_handler &c_handler, batch_handler &b_handler,
+	 auth_handler &a_handler)
```

Comment:
Can class_handler manage the check? It may be preferable to adding a new handler.